extensions/meduza/package.json

```
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "meduza",
+  "title": "Meduza",
+  "description": "Every day, we bring you the most important news and feature stories from hundreds of sources in Russia and across the former Soviet Union. Our team includes some of Russia's top professionals in news and reporting. We value our independence and strive to be a reliable, trusted outlet for verified, unbiased information about Russia and the former Soviet Union, as well as a source for sharp insights about one of the world's most enigmatic regions",
+  "icon": "icon.png",
+  "author": "castletaste",
+  "access": "public",
+  "categories": [
+    "News"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "en",
+      "title": "Meduza",
+      "subtitle": "News",
+      "description": "Latest daily news",
+      "mode": "view"
+    },
+    {
+      "name": "ru",
+      "title": "Meduza",
+      "subtitle": "Новости",
+      "description": "Последние новости",
+      "mode": "view"
+    },
```

```suggestion
      "name": "en",
      "title": "Show Latest News",
      "subtitle": "Meduza",
      "description": "Latest daily news",
      "mode": "view"
    },
```

Would be better to have only one command and change language inside the command, you can maybe use [list dropdown](https://developers.raycast.com/api-reference/user-interface/list#list.dropdown) for that

---

Yes, I also think that's more convenient